pkg/reconciler/openshift/namespace/namespace.go

```diff
@@ -175,6 +176,14 @@ func (ac *reconciler) admissionAllowed(ctx context.Context, req *admissionv1.Adm
 
 	logger.Infof("Trying to admit namespace: %s with SCC: %s", namespaceObject.Name, nsSCC)
 
+	securityClient := common.GetSecurityClient(ctx)
```

Comment:
can we use the verifySCCExist function here ?

Reply:
no, that exists in a separate package (tektonconfig validation) and not in this reconcile - it is small enough boilerplate that i didn't care to port it here too